examples/ktor/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+val ktor_version: String by project
+val logback_version: String by project
+
+description = "An Example GraphQL service served by a Spark HTTP server"
+
+plugins {
+    id("application")
+}
+
+application {
+    mainClassName = "io.ktor.server.netty.EngineMain"
+}
+
+val kotlinCoroutinesVersion: String by project
+dependencies {
+    implementation("io.ktor:ktor-server-core:$ktor_version")
+    implementation("io.ktor:ktor-server-netty:$ktor_version")
+    implementation("ch.qos.logback:logback-classic:$logback_version")
+
+    implementation("com.expediagroup", "graphql-kotlin-schema-generator")
+    implementation("org.jetbrains.kotlinx", "kotlinx-coroutines-jdk8", kotlinCoroutinesVersion)
+    testImplementation("org.slf4j", "slf4j-log4j12", "1.7.30")
```

Comment:
We can also add this version to the `gradle.properties` if we have the file

Reply:
just spotted it was not needed as logback is used anyway